packages/codec/lib/format/utils/inspect.ts

```diff
@@ -30,6 +30,12 @@ export interface ResultInspectorOptions {
    * the address.  (By default it displays only the ENS name.)
    */
   noHideAddress?: boolean;
+   /**
+   * This flag, if set, causes mappings to be rendered via objects
+   * rather than Maps.  This is intended for compatibility and not
+   * recommended for normal use.
+   */
+  renderMappingsViaObjects?: boolean;
```

Comment:
Should this be part of this PR? Is this a merge remnant?

Reply:
This was that bit of code I added so that the visual debugger could display mappings.  Possibly Tyler could pull it out into a separate PR or something, but it is here deliberately, its's not some artifact or something.

Reply:
I pulled Harry's work out into https://github.com/trufflesuite/truffle/pull/6074